webtool/test_api.py

```
@@ -433,6 +482,94 @@ def test_create_project_invalid_name_400(client):
+def test_jobstart_endpunkte_geriegelt_fuer_reservierte_projekte(client, tmp_path):
+    """Bot-Befund #491 (Minor, berechtigt): transcribe/correct/correct_file/
+    retranscribe ERZEUGEN den vergifteten Zeilenstrom selbst — ohne Riegel liefe
+    ein Altprojekt "active" weiter und fuelle gesehen/bases mit Muell (#478/#487).
+    Ein Altprojekt ist damit stillgelegt: lesen, umbenennen, loeschen geht,
+    kein neuer Lauf."""
+    for pfad in ("/api/projects/active/transcribe",
+                 "/api/projects/active/correct",
+                 "/api/projects/active/files/S1/correct",
+                 "/api/projects/active/files/S1/transcribe"):
+        r = client.post(pfad)
+        assert r.status_code == 400 and "reserviert" in r.json()["detail"], pfad
```

Ruff (0.16.2)

[warning] Assertion should be broken down into multiple parts

(PT018)